### DISCUSSION TITLE
Push notification with next pwa

### DISCUSSION THREAD

**Discussion by abdallahmoubarak** [DuCanhGH/next-pwa#152] [General]
Hello Guys,

how can I use push notification with next pwa?
and how to handle what user can receive notification?

**Comment by DuCanhGH** [comment]
@abdallahmoubarak you can use [our `web-push` example](https://github.com/DuCanhGH/next-pwa/tree/89e01b9a83c7e9131f83d7f442c72d8a70a76267/examples/web-push) as a reference.

Workbox also provides [a few blogs](https://web.dev/explore/notifications) on this. Have a look!

**Reply by abdallahmoubarak** [reply]
Thank you for sharing
